### ISSUE TITLE
Default Input style changed 2.3.0 -> 2.3.1

### ISSUE THREAD

**Issue by Relax594** [react-native-elements/react-native-elements#2537]
**Explain what you did (Required)**

Updated from 2.3.0 to 2.3.1

**Expected behavior (Required)**

Same default styling as in v2.3.0

**Describe the bug (Required)**

Default styling for the Input Component has changed going from 2.3.0 to 2.3.1

**To Reproduce (Required)**

We highly recommend that you re-create the bug on [Snack](https://snack.expo.io). If not, list the steps that a reviewer can take to reproduce the behaviour:

```bash
<Input
      ref={ref}
      editable={editable}
      inputContainerStyle={{
           borderBottomColor: 'transparent',
      }}
/>
```

**Screenshots (Required)**
v2.3.0
![grafik](https://user-images.githubusercontent.com/22765758/91689873-403a2c80-eb65-11ea-81b9-5fe4aed5011d.png)

v2.3.1 (FontSize seems to be smaller, its very clear on a device)
![grafik](https://user-images.githubusercontent.com/22765758/91689923-57791a00-eb65-11ea-9bc8-44be2000c169.png)

**Your Environment (Required):**

| software              | version |
| --------------------- | ------- |
| react-native-elements |  2.3.1  |
| react-native          |  0.63.2  |

**Comment by flyingcircle** [comment]
I was not able to reproduce this. I even went back and forth on the single line change in `Input` between 2.3.0 and 2.3.1. There is nothing in that release that would cause this, so I would likely look elsewhere for why this change happened. Perhaps something in your Theme caused this? If you can reproduce this in a snack, I'll reopen it.

**Comment by Relax594** [comment]
I know there is nothing in this release that could cause this, but it does. Just updating to 2.3.1 is enough, not changing anything beside that. The PR @JB1905 sent fixed that issue.

**Comment by Relax594** [comment]
@flyingcircle https://github.com/react-native-elements/react-native-elements/issues/2544 this is the same issue with 2.3.1

**Comment by JB1905** [comment]
The problem is connected with [this change](https://github.com/react-native-elements/react-native-elements/commit/06664f000dcce1925c251801059d139fcf0be2f6)

**Comment by flyingcircle** [comment]
Ahh! your original ticket didn't mention anything about `style={{}}` in order to force this style change. I'll keep this one closed and keep #2544 open to fix the issue.

**Comment by Relax594** [comment]
Yeah I didn't see that prop in the first place within our code^^ my mistake
